f2/src/forms/ContactInfoForm.js

```diff
@@ -55,17 +59,19 @@ export const ContactInfoForm = (props) => {
                     title={<Trans id="contactinfoPage.hasValidationErrors" />}
                   />
                 )}
-                {fyiForm ? (
+                <br />
+                <br />
+                {
```

Comment:
These curly braces and React.Fragment are not necessary if there is no longer conditional rendering.

Reply:
Advice taken, these <br /> and React.Fragment removed